--- lxqt-common-0.10.0/config/lxqt.conf.0000~	2015-10-08 18:38:12.000000000 +0200
+++ lxqt-common-0.10.0/config/lxqt.conf	2015-11-11 17:57:50.684549847 +0100
@@ -1,9 +1,12 @@
 [General]
-theme=frost
+theme=Light
 icon_theme=oxygen
 single_click_activate=false
 tool_button_style=ToolButtonTextBesideIcon
 
+[Mouse]
+cursor_theme=wonderland
+
 [Qt]
 font="Sans,11,-1,5,50,0,0,0,0,0"
 doubleClickInterval=400
--- lxqt-common-0.10.0/config/pcmanfm-qt/lxqt/settings.conf.in.0000~	2015-10-08 18:38:12.000000000 +0200
+++ lxqt-common-0.10.0/config/pcmanfm-qt/lxqt/settings.conf.in	2015-11-11 17:58:24.840362749 +0100
@@ -1,11 +1,11 @@
 [System]
 IconThemeName=elementary
-SuCommand=gksu %s
+SuCommand=pkexec %s
 TerminalCommand=
-Archiver=file-roller
+Archiver=ark
 SIUnit=false
-TerminalDirCommand=xterm
-TerminalExecCommand=xterm -e %s
+TerminalDirCommand=qterminal
+TerminalExecCommand=qterminal -e %s
 
 [Behavior]
 BookmarkOpenMethod=0
@@ -15,7 +15,7 @@ ConfirmDelete=true
 
 [Desktop]
 WallpaperMode=stretch
-Wallpaper=@LXQT_SHARE_DIR@/themes/frost/numix.png
+Wallpaper=/usr/share/wallpapers/mdk/default.jpg
 BgColor=#000000
 FgColor=#ffffff
 ShadowColor=#000000
--- lxqt-common-0.10.0/config/session.conf.0000~	2015-10-08 18:38:12.000000000 +0200
+++ lxqt-common-0.10.0/config/session.conf	2015-11-11 17:57:50.684549847 +0100
@@ -4,10 +4,12 @@ leave_confirmation=true
 
 [Environment]
 GTK_CSD=0
+BROWSER=qupzilla
+TERM=qterminal
 
 [Mouse]
 cursor_size=18
-cursor_theme=whiteglass
+cursor_theme=wonderland
 acc_factor=20
 acc_threshold=10
 left_handed=false
