diff -Naur lxqt-common-0.8.0/startlxqt.in lxqt-common-0.8.0.tpg/startlxqt.in
--- lxqt-common-0.8.0/startlxqt.in	2014-08-23 09:12:54.000000000 +0000
+++ lxqt-common-0.8.0.tpg/startlxqt.in	2015-01-12 23:09:50.124825391 +0000
@@ -1,9 +1,17 @@
 #!/bin/sh
 
+if [ -z "$XDG_DATA_HOME" ]; then
+    export XDG_DATA_HOME="$HOME/.local/share"
+fi
+
 if [ -z "$XDG_CONFIG_HOME" ]; then
 	export XDG_CONFIG_HOME="$HOME/.config"
 fi
 
+if [ -z "$XDG_DATA_DIRS" ]; then
+	export XDG_DATA_DIRS="$XDG_DATA_HOME:/usr/local/share/:/usr/share/"
+fi
+
 # Ensure the existance of the 'Desktop' folder
 if [ -e "$XDG_CONFIG_HOME/user-dirs.dirs" ]; then
 	. "$XDG_CONFIG_HOME/user-dirs.dirs"
